.goreleaser.yaml

```diff
@@ -101,7 +101,7 @@ brews:
       email: noreply@stackit.de
     homepage: "https://github.com/stackitcloud/stackit-cli"
     description: "A command-line interface to manage STACKIT resources.\nThis CLI is in a BETA state. More services and functionality will be supported soon."
-    folder: Formula
+    directory: Formula
```

Comment:
The previous field name was deprecated: https://goreleaser.com/deprecations#brewsfolder